src/action_cc.cc

```diff
@@ -896,3 +897,72 @@ NM_ACTION_(cmd_output) {
         ? nm_action_result_silent()
         : nm_action_result_msg("%s", qPrintable(Qt::convertFromPlainText(out, Qt::WhiteSpacePre)));
 }
+
+NM_ACTION_(nickel_bluetooth) {
+    enum BLUETOOTH_ACTION {
+        ENABLE  = 0b00001,
+        DISABLE = 0b00010,
+        TOGGLE  = 0b00100,
+        CHECK   = 0b01000,
+        SCAN    = 0b10000
+    };
+
+    int action = 0;
+    if (!strcmp(arg, "enable"))       action |= ENABLE;
+    else if (!strcmp(arg, "disable")) action |= DISABLE;
+    else if (!strcmp(arg, "toggle"))  action |= TOGGLE;
+    else if (!strcmp(arg, "check"))   action |= CHECK;
+    else if (!strcmp(arg, "scan"))    action |= SCAN;
+    else
+        NM_ERR_RET(nullptr, "unknown nickel_bluetooth action '%s'", arg);
+
+    //libnickel 4.34.20097 * _ZN16BluetoothManager14sharedInstanceEv
+    BluetoothManager *(*BluetoothManager_sharedInstance)();
+    NM_ACT_XSYM(BluetoothManager_sharedInstance, "_ZN16BluetoothManager14sharedInstanceEv", "could not dlsym BluetoothManager::sharedInstance");
+
+    //libnickel 4.34.20097 * _ZNK16BluetoothManager2upEv
+    uint (*BluetoothManager_up)(BluetoothManager *);
+    NM_ACT_XSYM(BluetoothManager_up, "_ZNK16BluetoothManager2upEv", "could not dlsym BluetoothManager::up");
+
+    //libnickel 4.34.20097 * _ZN16BluetoothManager2onEv
+    void (*BluetoothManager_on)(BluetoothManager *);
+    NM_ACT_XSYM(BluetoothManager_on, "_ZN16BluetoothManager2onEv", "could not dlsym BluetoothManager::on");
+
+    //libnickel 4.34.20097 * _ZN16BluetoothManager4scanEv
+    void (*BluetoothManager_scan)(BluetoothManager *);
+    NM_ACT_XSYM(BluetoothManager_scan, "_ZN16BluetoothManager4scanEv", "could not dlsym BluetoothManager::BluetoothManager::scanEv");
+
+    //libnickel 4.34.20097 *_ZN16BluetoothManager8stopScanEv
```

Comment:
Oh, oops, didn't notice the missed space here.

Reply:
My bad! Just sent the correction.